### ISSUE TITLE
Great project. Interfaced with telegram to control SP on Pi. Some issues with stop

### ISSUE THREAD

**Issue by rbnpi** [Widdershin/sonic-pi-cli#2]
This is a great little project. Easy to install and works well. I have interfaced it with telegram (see my project on http://rbnrpi.wordpress.com and it works well. Set to autoboot into telegram and sonic pi then controlled from my iPhone. Occasionally I find that the stop command only partially stops complex files, and so at present I issue it twice and it works fine.
Video of project in action coming to rbn1tube on you tube as soon as its processed.
http://youtu.be/9o-DY8qrm44

**Comment by Widdershin** [comment]
Awesome video!

Could you provide some of the complex music that `stop` doesn't quite stop? I haven't been able to reproduce this issue.

**Comment by rbnpi** [comment]
I looked at this again this morning. This time on my iMca. Stop workss fine there. I think the issue is that my B+ was pretty fully taxed on the demo film I did. When playing the earl of Essex Galliard, stop sonic-=Pi sometimes stopped one thread (right hand part) but left the other (left hand part running).
So I think the problem is mine not yours. It is "sovled" by sending the stop command twice in quick succession.
